.github/workflows/node.js.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         run: |
           yarn global add cross-env
           yarn
-          yarn compile:web
```

Comment:
@infinite-persistence @tzarebczan I was going nuts trying to figure out which of these changes could have caused error  `NODE_ENV environment variable is undefined` that was breaking the build.  However, I finally determined that this was actually a preexisting issue that has now been elevated from "warning" to "error" in webpack 5 (or, at least the environment plugin).

See here for the warning in a recent build from another ticket: https://github.com/OdyseeTeam/odysee-frontend/runs/7780999926?check_suite_focus=true#step:4:300

The `compile:web` script is used in `dev:web-server` as well as `run:web-server`, where `NODE_ENV` is set.  However, here it's used directly.  There's also the option of doing something like a `compile:web:prod` script that calls this with the environment variable set if you'd prefer that as well.